insonmnia/miner/network/l2tp_tuner.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/docker/go-plugins-helpers/ipam"
 	netDriver "github.com/docker/go-plugins-helpers/network"
 	log "github.com/noxiouz/zapctx/ctxlog"
+	"github.com/pkg/errors"
```

Comment:
Just errors.

Reply:
github.com/pkg/errors is much more usefull (not in this case though)